PackageDownloadsExample/PackageDownloadsExample.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NuGet.Protocol" Version="5.5.0-preview.1.6319" />
+    <PackageReference Include="System.CommandLine.DragonFruit" Version="0.3.0-alpha.20070.2" />
```

Comment:
This may make copy-pasting the sample a little harder. Might be simpler just to accept a single string arg